packages/neuron-ui/src/components/FormattedTokenAmount/index.tsx

```diff
@@ -8,25 +8,30 @@ import { HIDE_BALANCE } from 'utils/const'
 
 import styles from './formattedTokenAmount.module.scss'
 
-type FormattedTokenAmountProps = { item: State.Transaction; show: boolean }
+type FormattedTokenAmountProps = { item: State.Transaction; show: boolean; symbolClassName?: string }
 type AmountProps = Omit<FormattedTokenAmountProps, 'isNeedCopy'> & {
   sudtAmount: string
   isReceive: boolean
   amount: string
+  symbolClassName?: string
 }
 
-const Amount = ({ sudtAmount, show, item, isReceive, amount }: AmountProps) => {
+const Amount = ({ sudtAmount, show, item, isReceive, amount, symbolClassName }: AmountProps) => {
```

Comment:
It's a bit weird to pass `class name` for a sub-component, what if passing the sub-component directly

Reply:
`pass class name for a sub-component` makes it easy to change the sub-component styles.